drivers/clock_control/clock_control_nrf.c

```diff
@@ -613,6 +613,8 @@ static void clock_event_handler(nrfx_clock_evt_type_t event)
 	const struct device *dev = CLOCK_DEVICE;
 
 	switch (event) {
+#if !NRF_CLOCK_HAS_XO_TUNE
+	/* HFCLK started should be used only if tune operation is done implicitly. */
```

Comment:
Wouldn't it be a bit clearer if we put it this way:
```suggestion
#if NRF_CLOCK_HAS_XO_TUNE
	case NRFX_CLOCK_EVT_XO_TUNED:
		clkstarted_handle(dev, CLOCK_CONTROL_NRF_TYPE_HFCLK);
		break;
#else
	/* HFCLK started should be used only if tune operation is done implicitly. */
```